docs/CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -58,6 +58,7 @@ Download v1.10.4 in the [v1.10.4 release page](https://github.com/karmada-io/kar
 #### Others
 - The base image `alpine` now has been promoted from `alpine:3.20.1` to `alpine:3.20.2`. ([#5268](https://github.com/karmada-io/karmada/pull/5268))
 - Bump golang version to `v1.21.13` ([#5371](https://github.com/karmada-io/karmada/pull/5371), @zhzhuang-zju)
+- [Enable `karmada-data` flag in `karmadactl register` command](https://github.com/karmada-io/karmada/pull/5429) for storing karmada configuration files.
```

Comment:
You can add release notes at the location indicated in the diagram below. The changelog is released when a new release is made.

![image](https://github.com/user-attachments/assets/aa04a485-5c5c-4ded-87ca-bd863097ce84)